cmd/nodemarker/main.go

```diff
@@ -114,7 +129,7 @@ func main() {
 	})
 
 	startListeners := make(chan struct{})
-	if args.enableWebhook && !args.disableCertRotation {
+	if !args.disableCertRotation {
```

Comment:
I guess everything works because of the hardcoded `--disable-cert-rotation` in helm, but can we still keep checking that the webhooks should run (`!webhookDisabled` or `enabled || only`)?